libs/testing/src/user.session.ts

```diff
@@ -332,4 +334,14 @@ export class UserSession {
       await this.testAgent.post(`/v1/changes/${change._id}/apply`);
     }
   }
+
+  public async updateOrganizationServiceLevel(serviceLevel: ApiServiceLevelEnum) {
+    const organizationService = new OrganizationService();
+
+    await organizationService.updateServiceLevel(this.organization._id, serviceLevel);
+  }
+
+  public async updateEnvironmentApiRateLimits(apiRateLimits: Partial<IApiRateLimitMaximum>) {
+    await this.environmentService.updateApiRateLimits(this.environment._id, apiRateLimits);
+  }
```

Comment:
Adding methods to customise the organization service level and environment rate limit in tests.